config/containers/logging/local.md

```diff
@@ -1,46 +1,53 @@
 ---
-description: Describes how to use the local binary (Protobuf) logging driver.
-keywords: local, protobuf, docker, logging, driver
+description: Describes how to use the local logging driver.
+keywords: local, docker, logging, driver
 redirect_from:
 - /engine/reference/logging/local/
 - /engine/admin/logging/local/
-title: local binary file Protobuf logging driver
+title: Local File logging driver
 ---
 
-This `log-driver` writes to `local` binary files using Protobuf [Protocol Buffers](https://en.wikipedia.org/wiki/Protocol_Buffers)
```

Comment:
Is there a reason for removing the reference to the 3rd party file format? (maybe due to exclusive access comment below?)

Reply:
@darwin-traver I hope we can get some answers soon. :)

Reply:
Anusha has indicated that she will review this.